master.jinja2

```diff
@@ -20,6 +23,12 @@
 
 {% set enable_tests = enable_tests|default(true) %}
 
+{% set lava_device_type = lava_device_type|default("") %}
+{% set lava_job_name = lava_job_name|default("") %}
```

Comment:
Not sure why these two are needed

Reply:
they are needed for the L52 and L53.

Reply:
in all the other files its done like this:
{% block device_type %}x15{% endblock %}

I'm not convinced you need to set a variable to do this.

if you really want the variable, then you have to do the change in all the other files to and motivate why you do it.

Reply:
anyway, I updated to not change here to avoid affects on the existing files as possible as I can.